pom.xml

```diff
@@ -10,38 +10,36 @@
 
     <name>Restlet project</name>
     <description>Lightweight REST framework</description>
-    <url>http://www.restlet.org</url>
+    <url>https://restlet.talend.com/</url>
     <inceptionYear>2005</inceptionYear>
 
     <scm>
-        <url>http://restlet.tigris.org/svn/restlet/modules/</url>
+        <url>https://github.com/restlet/restlet-framework-java.git</url>
+        <connection>scm:git:${project.scm.url}</connection>
+        <developerConnection>scm:git:${project.scm.url}</developerConnection>
     </scm>
 
     <repositories>
```

Comment:
I think we can drop the `repositories` section. The`Restlet repository` is useful only for 2.4 and below versions of the the framework.

Reply:
I believe it should be stated here while you're still gonna publish there as well, right?

Reply:
oh, I have to tell you we want to publish to Maven Central for 2.5 versions and higher. That's why I'm inclined to drop the repositories section.
Does it sound correct to you?

Reply:
removed 👍